src/index.html

```diff
@@ -1,18 +1,31 @@
 <!doctype html>
-<html lang="en">
+<html lang="ru">
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Calendar</title>
+    <title>Календарь</title>
     <link
       rel="stylesheet"
       href="styles/index.scss"
     />
```

Comment:
The link tag should have a link to a CSS file, not SCSS. Web browsers can't interpret SCSS files directly, they need to be compiled into CSS.